TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -49,6 +49,7 @@ class RouteOptionsViewController: UIViewController {
     var searchTo: Place!
     var searchType: SearchBarType = .to
     var showRouteSearchingLoader: Bool = false
+    var trips: [Trip] = []
```

Comment:
What is this variable used for? I can't find any instances where it is written to or read in `RouteOptionsViewController`. In `updateAllRoutesLiveTracking(sender:)` you seem to recreate `trips` only locally.

Reply:
oops, this was from when I was considering rendering trips globally and not locally! I moved out where I render all the trips so now im using this variable as opposed to my local one.